src/EventSubscriber/BlameableSubscriber.php

```diff
@@ -108,7 +107,7 @@ public function preUpdate(LifecycleEventArgs $lifecycleEventArgs): void
         $oldValue = $entity->getUpdatedBy();
         $entity->setUpdatedBy($user);
 
-        $this->unitOfWork->propertyChanged($entity, self::UPDATED_BY, $oldValue, $user);
+        $this->entityManager->getUnitOfWork()->propertyChanged($entity, self::UPDATED_BY, $oldValue, $user);
```

Comment:
Could you extract `private function getUnitOfWork()` function, so we don't chained methods here?

Reply:
done 👍